packages/services/commerce/src/rate-limit/limiter.ts

```diff
@@ -294,7 +271,6 @@ export function createRateLimiter(config: {
         clearInterval(intervalHandle);
         intervalHandle = null;
       }
-      await (await postgres$).destroy();
```

Comment:
Why not this? It looks like postgres is still being used

Reply:
it's part of global shutdown in `index.ts`